target/scripts/helpers/relay.sh

```diff
@@ -179,24 +183,24 @@ function _setup_relayhost() {
     postconf "relayhost = ${DEFAULT_RELAY_HOST}"
   fi
 
-  if [[ -n ${RELAY_HOST} ]]; then
-    _log 'trace' "Setting up relay hosts (default: ${RELAY_HOST})"
+  _process_relayhost_configs
+}
 
-    _relayhost_sasl
-    _populate_relayhost_map
+# Called during initial container setup, or by change detection event:
+function _process_relayhost_configs() {
+  local DATABASE_RELAYHOSTS='/tmp/docker-mailserver/postfix-relaymap.cf'
 
-    _relayhost_configure_postfix
+  # One of these must configure a relayhost for the feature to relevant:
+  if [[ ! -f ${DATABASE_RELAYHOSTS} ]] \
+  && [[ -z ${DEFAULT_RELAY_HOST} ]] \
+  && [[ -z ${RELAY_HOST} ]]; then
+    return 1
   fi
```

Comment:
```suggestion
  if [[ ! -f ${DATABASE_RELAYHOSTS} ]] && [[ -z ${DEFAULT_RELAY_HOST} ]] && [[ -z ${RELAY_HOST} ]]; then
    return 1
  fi
```

I will make sure my setup works with these "long lines" 😆

Reply:
Personally the vertical arrangement is quicker to parse visually (minor), but it also fairs better for `git blame` history (also minor value in this context).

I can go with the single line if you want, did you want that to collapse the suggestion to a single line entirely with `&& return 1` like was suggested elsewhere?

Reply:
> I will make sure my setup works with these "long lines" 😆

Yeah, these old (VT100) terminals with a max line length of 80 are a PITA nowadays 🤣 You should really consider an upgrade in the near future.

Reply:
> But I have no objection to the current way either.

I choose the `<100` length multi-line split condition then 😝